sklearn/svm/_classes.py

```diff
@@ -1268,6 +1268,9 @@ class OneClassSVM(OutlierMixin, BaseLibSVM):
         vectors. Should be in the interval (0, 1]. By default 0.5
         will be taken.
 
+        .. versionchanged:: 0.26
+           The default value of ``nu`` will change to 0.1 in version 0.26.
```

Comment:
I think this `versionchanged` should be introduced prior to releasing 1.1 instead.